bitcoin-extensions/bitcoin.html

```diff
@@ -0,0 +1,33 @@
+<html>
+	<head>
+		<title>Bitcoin Price Index Watcher</title>
+		<link href="styles.css" rel="stylesheet" type="text/css">
+	</head>
+	<body>
+		<img id="logo" src="bitcoin.png" />
+		<div id="data"></div>
+		<script type="text/javascript">
```

Comment:
I love your usage of the javascript and making ajax requests. One way to keep everything more organized is to use an external javascript file. 

<script src="scripts.js"> </script>

Then write your javascripts in the scripts.js file. Generally, in coding, each file has one purpose.